options/posix/generic/posix_signal.cpp

```diff
@@ -131,3 +131,8 @@ int sigaltstack(const stack_t *__restrict ss, stack_t *__restrict oss) {
 
 	return 0;
 }
+
+int sigisemptyset(const sigset_t *) {
+	__ensure(!"sigpending() not implemented");
```

Comment:
Wrong message, also this is easy to implement (`return !(*set);` or so)

Reply:
Should be fixed.